src/transactionRequests/routes.js

```diff
@@ -43,21 +43,10 @@ module.exports = [
       description: 'Get a transaction request by ID',
       validate: {
         headers: Joi.object({
-          accept: Joi.string().optional().regex(regexAccept),
-          'content-type': Joi.string().required().regex(regexContentType),
-          date: Joi.date().format('ddd, D MMM YYYY H:mm:ss [GMT]').required(),
-          'x-forwarded-for': Joi.string().optional(),
-          'fspiop-source': Joi.string().required(),
-          'fspiop-destination': Joi.string().optional(),
-          'fspiop-encryption': Joi.string().optional(),
-          'fspiop-signature': Joi.string().optional(),
-          'fspiop-uri': Joi.string().optional(),
-          'fspiop-http-method': Joi.string().optional(),
-          traceparent: Joi.string().optional(),
-          tracestate: Joi.string().optional()
+          accept: Joi.string().optional().regex(regexAccept)
```

Comment:
Please leave those in.. most of them are in the Swagger (the trace related headers are for supporting tracing)

Reply:
You're right @elnyry. In this instance I was looking at the swagger file in trx-req-svc and which does not have those headers.